base/commandLine.cpp

```diff
@@ -1143,9 +1144,15 @@ void upgradeTargets() {
 		// ScummVM still generates an incorrect description string. So, the description
 		// should only be updated if the user explicitly requests this.
 #if 0
+<<<<<<< HEAD
```

Comment:
Oops, this looks like it needs resolving :-)

Reply:
Thanks, fixed!